app/models/role.rb

```diff
@@ -1,6 +1,9 @@
 # frozen_string_literal: true
 
 class Role < ApplicationRecord
+  alias_attribute :tag, :role
```

Comment:
The reason for this is that I chose a poor name for the 'role' field on the role, i.e. the internal identifier for the role.  In some queries, seeing `role.role` made it difficult to understand.  I can change this properly if it's considered necessary